src/components/updatemenus/attributes.js

```diff
@@ -11,6 +11,7 @@
 var fontAttrs = require('../../plots/font_attributes');
 var colorAttrs = require('../color/attributes');
 var extendFlat = require('../../lib/extend').extendFlat;
+var padAttrs = require('../../plots/pad_attributes');
```

Comment:
That's where `pad_attributes.js` at the moment, but I hate to see see files in `src/plots/` root.

Maybe we add a `src/plots/common/`folder? or `src/components/common/`? or even a `src/common/` for similar shared modules.